DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/userpreferences/file.directive.js

```diff
@@ -0,0 +1,17 @@
+﻿(function () {
+    'use strict';
+
+    angular
+    .module('app.userpreferences')
+    .directive('customOnChange', file);
+
+    function file() {
+        return {
+            restrict: 'A',
+            link: function (scope, element, attrs) {
+                var onChangeHandler = scope.$eval(attrs.customOnChange);
```

Comment:
Ok, let it be so. I propose to add here comment with the copyright
http://stackoverflow.com/questions/17922557/angularjs-how-to-check-for-changes-in-file-input-fields